### PR TITLE
2510 create Fault Tolerance 1.1 feature

### LINKED ISSUE #2510: FT: Create mpFaultTolerance-1.1 feature

### PR DESCRIPTION
**Pull request by tevans78**
fixes #2510 
#build

### CODE REVIEW

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/mpFaultTolerance-1.1/resources/l10n/com.ibm.websphere.appserver.mpFaultTolerance-1.1.properties [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature provides support for the MicroProfile Fault Tolerance 1.1 API for enterprise Java.
```
Comment (bsbyrd1) [inline dev/com.ibm.websphere.appserver.features/visibility/public/mpFaultTolerance-1.1/resources/l10n/com.ibm.websphere.appserver.mpFaultTolerance-1.1.properties]: You must capitalize Enterprise Java.

Comment (Azquelt) [reply dev/com.ibm.websphere.appserver.features/visibility/public/mpFaultTolerance-1.1/resources/l10n/com.ibm.websphere.appserver.mpFaultTolerance-1.1.properties]: The FaultTolerance-1.0 has it non-capitalized as well. Should we go back and change it there too?

Comment (Azquelt) [reply dev/com.ibm.websphere.appserver.features/visibility/public/mpFaultTolerance-1.1/resources/l10n/com.ibm.websphere.appserver.mpFaultTolerance-1.1.properties]: Interestingly, mpConfig-x.x doesn't mention Enterprise Java and microprofile-1.3 says it's for Cloud Native Java.

**Comment by Azquelt** [comment]
#build

**Comment by Azquelt** [comment]
#build

**Comment by Azquelt** [comment]
@bsbyrd1 if that's ok I think we're good to merge.

**Pull request merged**